src/pages/tasks/[id].tsx

```diff
@@ -11,6 +14,16 @@ const TaskDetailsPage = () => {
         return <PageNotFound />;
     }
 
-    return <TaskDetails url={TASK_DETAILS_URL} taskID={id} />;
+    const { data } = useGetTaskDetailsQuery(id);
+    const taskDetails: any = data?.taskData;
+    const taskURL: any = `https://status.realdevsquad.com/tasks/${id}`;
+
+    return (
+        <>
+            <LinkPreviewCard taskDetails={taskDetails} taskURL={taskURL} />
```

Comment:
Since LinkPreviewCard is a separate component
Can we look into naming just url here for consistency

Reply:
Thanks for reviewing, will update them soon